packages/fxa-content-server/app/scripts/templates/ready.mustache

```diff
@@ -16,23 +16,17 @@
     {{/isSync}}
 
     {{^isSync}}
-      {{#serviceName}}
+      {{#emailVerified}}
```

Comment:
@shane-tomlinson I took out the `serviceName` checks in other template files since we don't need to check for truthiness anymore and reversed the logic here to make sure we display `emailVerified` if it's truthy and display the `serviceName` section here otherwise.

Reply:
Something about this change seems off, because the button on line 26 should only show to OAuth RPs if the account is verified.

Reply:
After looking at ready.js (and being very confused), `escapedEmailReadyText` is only displayed when a user has to verify their primary/secondary email for making changes to the account settings. 

Now we need to disentangle the rest of the branches. I really don't like mustache sometimes.

Maybe something like this?
```mustache
{{#escapedEmailReadyText}}
  <p class="account-ready-service">{{{escapedEmailReadyText}}}</p>
{{/escapedEmailReadyText}}
{{^escapedEmailReadyText}}
  {{#emailVerified}}
    <p class="account-ready-service">{{#t}}You are now ready to use %(serviceName)s{{/t}}</p>
    {{#showContinueButton}}
      <div class="button-row">
        <button class="primary-button btn-continue">{{#t}}Continue{{/t}}</button>
      </div>
    {{/showContinueButton}}
  {{/emailVerified}}
  {{^emailVerified}}
    <p class="account-ready-generic">{{#t}}Your account is ready!{{/t}}</p>
  {{/emailVerified}}
{{/escapedEmailReadyText}}
```

Reply:
Sorry in advance for the length of this comment but let me tell you how I got from your suggestion to what I just pushed.

Did you try running unit tests with your suggestion? I don't think we can use escaped HTML to check for truthiness. I battled too long with this today:
```
TypeError: string is undefinedinterpolate
```
Seen at: http://127.0.0.1:3030/tests/index.html?grep=views%2Fready

I tried temporarily changing `escapedEmailReadyText` to a string to make sure that check is what's causing that error, and one test fails with a different error. I also don't see where we check for escaped text like this in other mustache files (I searched the project for `{{{` and we always use a different variable check when conditionally rendering escaped text).

This of course doesn't solve the problem, but I'd asked Ryan about that "Your account is ready!" text a few days ago, and he said he wants it replaced with "You are now ready to use Account Settings". I changed what you suggested to this:

```
{{#escapedEmailReadyText}}
  <p class="account-ready-service">{{{escapedEmailReadyText}}}</p>
{{/escapedEmailReadyText}}
{{^escapedEmailReadyText}}
  <p class="account-ready-service">{{#t}}You are now ready to use %(serviceName)s{{/t}}</p>
  {{#emailVerified}}
    {{#showContinueButton}}
      <div class="button-row">
        <button class="primary-button btn-continue">{{#t}}Continue{{/t}}</button>
      </div>
    {{/showContinueButton}}
  {{/emailVerified}}
{{/escapedEmailReadyText}}
```

... I then tried creating `showEscapedEmailReadyText` as a boolean of `escapedEmailReadyText`, but it looks like `escapedEmailReadyText` is always truthy anyway. 🤦‍♀️ Since we can't use handlebars `{{#if}}`, I needed a way to check for if there was an RP or not, which was previously done with a check for `serviceName`.

That's how I ended up here. I stuck `'Account Settings'` as a string in `Constants.js` so we can update that text in one place and our `if` check in `ready.js` for `isFromRelyingParty` will still work, but I left the tests as-is (except for the `service-mixin` test) since I feel those should probably be explicit. I reverted back to what we had in `ready.mustache` before modifications, and changed `serviceName` checks to `isFromRelyingParty`, and updated "Your account is ready!" to "You are now ready to use Account Settings". We lose a little reuse here since lines 20 and 33 are the same, but, I think that's fine.

Reply:
> This of course doesn't solve the problem, but I'd asked Ryan about that "Your account is ready!" text a few days ago, and he said he wants it replaced with "You are now ready to use Account Settings". I changed what you suggested to this:

Unfortunately, a user that opens a verification reminder link in a browser other than the  original browser they signed up in will not be able to be redirected to settings because they won't have a session. Is that handled by one of the other cases? If so, then all good.

Reply:
@shane-tomlinson I'm not sure the user would have had the option to redirect to Account Settings anyway because they wouldn't be shown the "Continue" button. 🤔

I went ahead and reverted that change back to the generic "Your account is ready!" copy... but now I'm wondering if it would actually be OK to display it without the redirect button. @ryanfeeley this goes back to my Slack convo with you about this "ready" page. If we should display a "Continue" button with "You are now ready to use Account Settings," on this screen, I can create another issue for that so we can get this in which will show "Your account is ready!", or, if I understand Shane's comment and all of the different conditionals here, I think we can display "You are now ready to use Account Settings" without it.

Reply:
Per @ryanfeeley, he would prefer "You are now ready to use Account Settings" without the redirect button instead of "Your account is ready!"

I'll change this back later today. I can open a new issue around displaying/getting the continue button to work.